Bigtable/src/Admin/V2/Gapic/BigtableInstanceAdminGapicClient.php

```diff
@@ -171,29 +179,24 @@ private static function getClientDefaults()
     {
         return [
             'serviceName' => self::SERVICE_NAME,
-            'serviceAddress' => self::SERVICE_ADDRESS,
-            'port' => self::DEFAULT_SERVICE_PORT,
-            'scopes' => [
-                'https://www.googleapis.com/auth/bigtable.admin',
-                'https://www.googleapis.com/auth/bigtable.admin.cluster',
-                'https://www.googleapis.com/auth/bigtable.admin.instance',
-                'https://www.googleapis.com/auth/bigtable.admin.table',
-                'https://www.googleapis.com/auth/cloud-bigtable.admin',
-                'https://www.googleapis.com/auth/cloud-bigtable.admin.cluster',
-                'https://www.googleapis.com/auth/cloud-bigtable.admin.table',
-                'https://www.googleapis.com/auth/cloud-platform',
-                'https://www.googleapis.com/auth/cloud-platform.read-only',
-            ],
-            'clientConfigPath' => __DIR__.'/../resources/bigtable_instance_admin_client_config.json',
-            'restClientConfigPath' => __DIR__.'/../resources/bigtable_instance_admin_rest_client_config.php',
+            'serviceAddress' => self::SERVICE_ADDRESS.':'.self::DEFAULT_SERVICE_PORT,
+            'clientConfig' => __DIR__.'/../resources/bigtable_instance_admin_client_config.json',
             'descriptorsConfigPath' => __DIR__.'/../resources/bigtable_instance_admin_descriptor_config.php',
-            'versionFile' => __DIR__.'/../../VERSION',
+            'credentialsConfig' => [
+                'scopes' => self::$serviceScopes,
+            ],
+            'transportConfig' => [
+                'rest' => [
+                    'restClientConfigPath' => __DIR__.'/../resources/bigtable_instance_admin_rest_client_config.php',
+                ],
+            ],
+            'versionFile' => __DIR__.'/../../../VERSION',
```

Comment:
It looks like Bigtable and Spanner's admin `versionFile`s should be listed one level further up.

Reply:
Fixed. We should discuss offline about how we want to handle this key.

Reply:
Sounds good